src/js/comments.js

```diff
@@ -0,0 +1,117 @@
+      },
+      body: JSON.stringify(newData),
+    },
+  );
+  return response.text();
+};
+
+const submitComment = (id, form) => {
+  form.addEventListener('submit', (e) => {
+    const inputName = document.querySelector('.input-name');
+    const inputComment = document.querySelector('.input-comment');
+    const commentN = document.querySelector('.commentN');
+    const num = parseInt(commentN.innerHTML, 10) + 1;
+    commentN.innerHTML = num || '';
+    const newComment = {
+      item_id: id,
+      username: inputName.value,
+      comment: inputComment.value,
+    };
+    e.preventDefault();
+    sendComment(newComment);
+  });
+};
+
+const URL = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/0tY3ZECsTgObPiElFJSy/comments?item_id=';
+const getComments = async (id) => {
+  const res = await fetch(`${URL}${id}`);
+  const comments = await res.text();
+  return comments;
+};
+
+const popUp = async (li) => {
+  const popDiv = document.createElement('section');
+  popDiv.className = 'popUp';
+  const movies = await makeShowUrl(li.id);
+  const response = await getComments(li.id);
+  const comments = JSON.parse(response);
+  const commentNum = commentCounter(comments);
+  const lastComment = comments[commentNum - 1] || [];
+
+  popDiv.innerHTML = `
+     <div class="w-100 container">
+         <i class="cancel-pop fa fa-times"></i>
+        <div class="w-100 container d-flex justify-content-center align-items-center flex-column">
+          <div class="pop-summary d-flex justify-content-between w-100">
+             <div class="pop-img pr-5">
+              <img class="w-100 h-100" src="${movies.image.original}" />
+            </div>
+          <div class="sam-p">
+          <h6>Summary: </h6>
+            ${movies.summary}
+          </div>
+
+          </div>
+          <div class="pop-info py-3 d-flex  w-100 flex-column">
+            <h2 class="mx-auto">${movies.name}</h2>
+            <div class="d-flex w-100 justify-content-between">
+              <div>
+              <span>Genres :   ${movies.genres
+    .map((movie) => `<span>${movie}</span>`)
+    .join(', ')}</span>              
+              </div>
+              <div>language: ${movies.language}</div>
+            </div>
+            <div class="d-flex justify-content-between">
+                <div>
+                  Schedule : ${movies.schedule.days}, ${movies.schedule.time} 
+                </div>
+                <div>
+                  TV: ${movies.network.name}, ${movies.network.country.name}
+                </div>
+            </div>
+          </div>
+
+          <div class="all-comments">
+                <span>Comments</span> 
+                <span class="text-secondery mx-2" ><i class="fa fa-comments" ></i></span> 
+                <span class="commentN text-white">${commentNum || ''}</span>
+                <p class="last-com font-italic">Last comment: ${
+  lastComment.creation_date || 'No comment'
+} </p>
```

Comment:
- [ ] As per the requirements of the project, When the `Comments popup` loads, the web app is expected to retrieve data from the selected API and show details about the selected item. And likewise, retrieve data from the [Involvement API](https://www.notion.so/Involvement-API-869e60b5ad104603aa6db59e08150270) to show the item comments. Now we have only updated the number of comments. We should display the comments also. Let's try to fix this issue.

yours
![image](https://user-images.githubusercontent.com/53019916/129229806-827120e2-5b5c-44de-90b3-c75cdd2dc3fb.png)

Original example
![image](https://user-images.githubusercontent.com/53019916/129229940-95e19f2f-feee-4df6-8f61-62b555f4c625.png)